backend/src/main/java/com/example/worksync/controller/ProjectController.java

```diff
@@ -43,4 +43,11 @@ public ResponseEntity<Void> deleteProject(@PathVariable Long id) {
         projectService.deleteProject(id);
         return ResponseEntity.noContent().build();
     }
+
+    @PostMapping("/{projectId}/addParticipant/{userId}")
```

Comment:
trocar o nome da rota para:

```suggestion
    @PostMapping("/{projectId}/participants/{userId}")
```